sdk/internal/recording/matchers.go

```diff
@@ -0,0 +1,39 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package recording
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+)
+
+// Optional parameters for the AddBodilessMatcher operation
+type MatcherOptions struct {
+}
+
+// AddBodilessMatcher adjusts the "match" operation to exclude the body when matching a request to a recording's entries.
+func AddBodilessMatcher(t *testing.T, options *MatcherOptions) error {
+	if recordMode != PlaybackMode {
+		return nil
+	}
+	req, err := http.NewRequest("POST", "http://localhost:5000/Admin/SetMatcher", http.NoBody)
+	if err != nil {
+		panic(err)
+	}
+	req.Header["x-abstraction-identifier"] = []string{"BodilessMatcher"}
+	req.Header["x-recording-id"] = []string{GetRecordingId(t)}
```

Comment:
Does this mean BodilessMatcher applies only to this recording?

Reply:
Yes, going to upgrade so you can pass in `nil` for the `t` argument and will apply to all (which I think is a less common use case).